src/dreyfus_fabric_info.erl

```diff
@@ -19,15 +19,20 @@
 -include_lib("mem3/include/mem3.hrl").
 -include_lib("couch/include/couch_db.hrl").
 
--export([go/3]).
+-export([go/4]).
 
-go(DbName, DDocId, IndexName) when is_binary(DDocId) ->
+go(DbName, DDocId, IndexName, InfoLevel) when is_binary(DDocId) ->
     {ok, DDoc} = fabric:open_doc(DbName, <<"_design/", DDocId/binary>>, []),
-    go(DbName, DDoc, IndexName);
+    go(DbName, DDoc, IndexName, InfoLevel);
 
-go(DbName, DDoc, IndexName) ->
+go(DbName, DDoc, IndexName, InfoLevel) ->
     Shards = mem3:shards(DbName),
-    Workers = fabric_util:submit_jobs(Shards, dreyfus_rpc, info, [DDoc, IndexName]),
+    case InfoLevel of
+        info ->
+            Workers = fabric_util:submit_jobs(Shards, dreyfus_rpc, info, [DDoc, IndexName]);
```

Comment:
That's what I started out with but I thought this would be more readable. Will switch it back